xocto/localtime.py

```diff
@@ -729,3 +730,13 @@ def period_exceeds_one_year(start_at: datetime_.datetime, end_at: datetime_.date
         one_year_after_start_at = one_year_after_start_at.replace(month=3, day=1)
 
     return tz_unaware_end_at > one_year_after_start_at
+
+
+def parse_date(value: str) -> datetime_.date:
+    _datetime = date_parser.parse(value, dayfirst=("/" in value))
+    return _datetime.date()
+
+
+def parse_dt(value: str) -> datetime_.datetime:
+    _datetime = date_parser.parse(value, dayfirst=("/" in value))
+    return timezone.make_aware(_datetime)
```

Comment:
Surprised we're not just using `.from_isoformat` here to avoid ambiguity?

Reply:
Might just ping @leamingrad for his view.

The code looks good though.

Reply:
> Surprised we're not just using .from_isoformat here to avoid ambiguity?

That's tempting as a first pass - I actually didn't realise e.g. `2020-02-02 10:00` is valid ISO (thought it had to have a `T` marker). I don't mind really - would like to keep "parse_dt" naming though as it's shorter.

Reply:
I agree - I would probably just use `datetime.datetime.from_isoformat`.

The only issue is that it doesn't accept the `Z` timezone in Python 3.10, but you can work around that with string replacement.

Reply:
Fair enough: updated to `from_isoformat` 👍. We can always extend it if we actually want to support more parseable formats, but I've not seen a need for this just yet - and the tests I added still pass ✅.

> The only issue is that it doesn't accept the Z timezone

Shouldn't matter: `parse_dt` expects a naive timestamp anyway.